### PR TITLE
fix: wrong log format for splunk-hec-logging

### PR DESCRIPTION
**Pull request by jiangfucheng**
### Description

Fixes https://github.com/apache/apisix/issues/9458

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by monkeyDluffy6017** [comment]
@Sn0rt please help to review

**Comment by Sn0rt** [comment]
LGTM

File: t/plugin/splunk-hec-logging.t [hunk 1]
```diff
@@ -375,3 +375,50 @@ hello world
...
+                upstream = {
+                    type = "roundrobin",
+                    nodes = {
+                        ["127.0.0.1:1980"] = 1
+                    }
+                },
+                plugins = {
+                    ["splunk-hec-logging"] = {
+                        endpoint = {
+                            uri = "http://127.0.0.1:18088/services/collector",
```
Comment (Sn0rt) [inline t/plugin/splunk-hec-logging.t:397]: where is the backend server ?

Comment (Revolyssup) [reply t/plugin/splunk-hec-logging.t:397]: It is run in CI https://github.com/apache/apisix/blob/1e31e9da12191789ea30439e82d2d45ed50b9b00/ci/pod/docker-compose.plugin.yml#L195

Comment (Revolyssup) [reply t/plugin/splunk-hec-logging.t:397]: https://github.com/apache/apisix/blob/1e31e9da12191789ea30439e82d2d45ed50b9b00/ci/pod/docker-compose.plugin.yml#L195

Comment (Sn0rt) [reply t/plugin/splunk-hec-logging.t:397]: copy

**Review by Sn0rt — APPROVED** [review]

**Review by monkeyDluffy6017 — DISMISSED** [review]

**Comment by monkeyDluffy6017** [comment]
The real test cases are needed and i see an issue has been created: https://github.com/apache/apisix/issues/9489.
So it's enough for this pr

**Comment by monkeyDluffy6017** [comment]
Hi @jiangfucheng, It looks like that some ci failed because of your commit, please fix the ci error

**Comment by jiangfucheng** [comment]
> Hi @jiangfucheng, It looks like that some ci failed because of your commit, please fix the ci error

Fixed, thx.

**Review by monkeyDluffy6017 — APPROVED** [review]

**Comment by Sn0rt** [comment]
@leslie-tsang pls help review.

**Review by Sn0rt — APPROVED** [review]

**Review by Sn0rt — APPROVED** [review]

**Review by soulbird — APPROVED** [review]

**Review by leslie-tsang — APPROVED** [review]

**Pull request merged**